app/models/encryptable.rb

```diff
@@ -47,16 +46,37 @@ def team
     folder.team
   end
 
+  def recrypt_transferred(receiver_private_key, team_password)
+    decrypt(plaintext_transfer_password(receiver_private_key))
+    self.encrypted_transfer_password = nil
+    encrypt(team_password)
+    save!
+  end
+
+  def transferred?
+    encrypted_transfer_password.present? && sender_id.present?
+  end
+
+  def inbox_folder_present?
+    Folder.find(folder_id)&.name == 'inbox' if folder_id
+  end
+
+  def plaintext_transfer_password(private_key)
+    Crypto::Rsa.decrypt(encrypted_transfer_password, private_key)
+  end
+
   private
 
   def encrypt_attr(attr, team_password)
     cleartext_value = send(:"cleartext_#{attr}")
 
-    encrypted_value = if cleartext_value.blank?
+    # rubocop:disable all
+    encrypted_value = if cleartext_value.nil? || cleartext_value.empty?
```

Comment:
Needs to be done like this because when using .blank? or &.blank? on transfer from images and pdf's, then we get this error:
![image](https://user-images.githubusercontent.com/68543649/235471415-1b4e5780-8e7f-4ec2-b626-3e8819e50202.png)